esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2490,32 +2490,30 @@ bool CFileSprayEx::onCopy(IEspContext &context, IEspCopy &req, IEspCopyResponse
         StringBuffer fileMask;
         constructFileMask(destTitle.str(), fileMask);
 
+        Owned<IUserDescriptor> udesc=createUserDescriptor();
         const char* srcDali = req.getSourceDali();
-        bool supercopy = req.getSuperCopy();
-        if (supercopy)
+        const char* srcu = req.getSrcusername();
+        if (srcDali && !isEmptyString(srcu))
```

Comment:
If 'srcu' could be "" (and hence isEmptyString()), then can't 'srcDali' also be?

Reply:
I will check.